src/components/Crm-add-new-product-button/CrmAddNewProductButton.tsx

```diff
@@ -7,22 +7,49 @@ import {
   newProductPriceSchema
 } from '../../helpers/validateNewProduct'
 import styles from './crmAddNewProductButton.module.scss'
-import { useEffect } from 'react'
-import {
-  addPrice,
-  createNewProduct
-} from '../../redux/crm-add-new-product/operation'
+import { FC, useEffect, useState } from 'react'
+import { addPrice, createNewProduct } from '../../redux/crm-product/operation'
 import { AppDispatch } from '../../redux/store'
-import { setFormErrors } from '../../redux/crm-add-new-product/slice/product'
-import { ProductStatus } from '../../types'
+import { setFormErrors } from '../../redux/crm-product/createSlice/product'
+import { ProductResponse, ProductStatus } from '../../types'
 import { RootState } from '../../redux/store/index'
+import { useNavigate, useParams } from 'react-router-dom'
+import axiosInstance from '../../axios/settings'
+import { Report } from 'notiflix/build/notiflix-report-aio'
+import { AxiosError } from 'axios'
+
+type Props = {
+  product: ProductResponse | undefined
+}
+
+const CrmAddNewProductButton: FC<Props> = () => {
```

Comment:
you have props but do not use props at all

Reply:
done